### ISSUE TITLE
Customize the working directory

### ISSUE THREAD

**Issue by sestrella** [stackbuilders/hapistrano#149]
Having a way to customize the working directory is useful for scenarios where
different components of an app reside inside the same repository (monorepo),
many CI allows to customize the working directory before running a set of steps
making each line less verbose. Let's consider a repo with the following structure:

```
backend/
frontend/
```

Right now in order to run a task inside a specific folder is it necessary to
perform a `cd` before each command:

```yml
build_script:
  - cd backend && stack setup
  - cd backend && stack build
```

The previous configuration could be rewritten as follow by adding a
`working_directory` parameter in the configuration file:

```yml
...
working_directory: backend
build_script:
  - stack setup
  - stack build
```

**Comment by CristhianMotoche** [comment]
Hey @sestrella Was this issue fixed successfully by #155 ?

**Comment by sestrella** [comment]
@CristhianMotoche yes, I think this ticket is solved by #155
